v3/1upkeyboards/1upslider8/1upslider8.json

```diff
@@ -0,0 +1,132 @@
+{
+    "name": "1upslider8",
+    "vendorId": "0x6F75",
+    "productId": "0x5611",
+    "matrix": { "rows": 1, "cols": 9},
+    "keycodes": ["qmk_lighting"],
+    "menus": [
```

Comment:
could use `"menus": ["qmk_rgb_matrix"]` since you have all effects enabled?

Reply:
Would that be instead of the current menu structure I have? One of the main reasons I have everything listed out explicitly is to make some small modifications to the adjustments on specific effects.

Reply:
Ok thats fine if you wanted some adjustments.